crawl/bazel.py

```diff
@@ -95,6 +96,34 @@ def query_all_libraries(repository_root_path, packages, verbose=False):
 def _normalize_dependency_string_to_group_id_artifact_id(dependency):
     return ":".join(dependency.split(":")[:2]) # remove classifier/package, so we can grab version from artifacts list
 
+def parse_override_file(file_path):
+    """
+    Returns a dict of {dep: overridded_dep} mapping
+    """
+    with open(file_path, "r") as f:
+        contents = f.read()
+
+        # Removes comments
+        contents = re.sub("#.*z*\n", "", contents).split("{")[1].split("}")[0]
```

Comment:
What is the z here for? Don't think it's needed.

Reply:
Yeah, as we discussed "#.*\n" should also work in the same way. I will try with regex once and if it works then will remove the z* part.